Configure

```diff
@@ -285,6 +285,7 @@ my @disablables = (
     "ocsp",
     "poly1305",
     "posix-io",
+    "pqueue",
```

Comment:
I don't think this actually does anything, since pqueue was merged into `ssl/` and there's no "NO_PQUEUE" define.